src/util/time.h

```diff
@@ -6,9 +6,11 @@
 #ifndef BITCOIN_UTIL_TIME_H
 #define BITCOIN_UTIL_TIME_H
 
+#include <chrono>
 #include <stdint.h>
 #include <string>
-#include <chrono>
+
+using namespace std::chrono_literals;
```

Comment:
trivia, both this and `using namespace std::literals::chrono_literals;` appear to be valid